functions/commonfn.php

```diff
@@ -357,6 +361,33 @@ function amuRegisterUser(){
 		//return message
 		return $newid;
 	}
+
+    /**
+     * Contacts central auth to determine if uun exists
+     *
+     * @param string $uun
+     *
+     * @return boolean
+     */
+    function is_valid_uun( $uun ) {
+        global $amu_ldap_config;
+        $ldapconn  = ldap_connect(  $amu_ldap_config['ldaphost'],  $amu_ldap_config['ldapport'] ) or wp_die( "Could not connect to {$amu_ldap_config['ldaphost']}", 200 );
+        $filter    = '(&(uid=' . $uun . '))';
+        $justthese = array( '*' );
```

Comment:
Is there a more expressive name we can use for this variable? I'm not sure what is is representing in its current form. It would be good to change this to `$just_these` if we are sticking with this name. Thanks Richard.

Reply:
Awesome, thanks Richard!